### PR TITLE
[MIG][16.0] stock_picking_completion_info

### PR DESCRIPTION
**Pull request by glitchov**
Migration from 14.0

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# https://github.com/OCA/stock-logistics-warehouse/pull/1566
+odoo-addon-stock-move-common-dest @ git+https://github.com/acsone/stock-logistics-warehouse.git@16.0-mig-stock_move_common_dest-hda#subdirectory=setup/stock_move_common_dest
```
Comment (rousseldenis) [inline test-requirements.txt]: @glitchov Please put this in a separate commit to easy removal.

Comment (rousseldenis) [reply test-requirements.txt]: Could you please also add the depending PR on this PR description in order to easy reviewers job ? Thanks

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

**Review by lmignon — APPROVED** [review]
LGTM (Code reveiw)

Thank you @glitchov
